sites/fast-website/src/public/index.html

```diff
@@ -131,7 +131,57 @@ <h3>AngularJS (1.x) 1.7.9</h3>
+                <p>Out of the box, FAST components are built on React, but you can build components on any modern framework using the FAST system.</p>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">View Github</fast-anchor>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">Read Documentation</fast-anchor>
+              </feature-card>
+              <feature-card>
+                <h5 slot="header">04</h5><h4 slot="header">Web Standards</h4>
+                <p>All FAST components follow WCAG 2.1, are W3C spec-compliant and use the W3C interaction models when available.</p>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">View Github</fast-anchor>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">Read Documentation</fast-anchor>
+              </feature-card>
+              <feature-card>
+                <h5 slot="header">05</h5><h4 slot="header">Animation</h4>
+                <p>Design sophisticated animation sequences with the animation library, an interface for the Web Animations API.</p>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">View Github</fast-anchor>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">Read Documentation</fast-anchor>
+              </feature-card>
+              <feature-card>
+                <h5 slot="header">06</h5><h4 slot="header">Colors</h4>
+                <p>Create color palettes, extract colors from images, and handle other color operations using our color library.</p>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">View Github</fast-anchor>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">Read Documentation</fast-anchor>
+              </feature-card>
+              <feature-card>
+                <h5 slot="header">07</h5><h4 slot="header">Layout / Grid</h4>
+                <p>Build layouts such as a 12 column grid for content or an application grid with resizable panels.</p>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">View Github</fast-anchor>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">Read Documentation</fast-anchor>
+              </feature-card>
+              <feature-card>
+                <h5 slot="header">08</h5><h4 slot="header">Other</h4>
+                <p>Leverage a toolkit of general utilities such as keyboarding, Right-To-Left (RTL), number, and string manipulation.</p>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">View Github</fast-anchor>
+                <fast-anchor slot="footer" class="link" href="#" appearance="lightweight">Read Documentation</fast-anchor>
+							</feature-card>
+						</card-section>
```

Comment:
I noticed that there were other comments about spacing, and thought I would put a note here so you saw it was present here as well.

Reply:
I think I have it fixed now, although I have no idea what actually caused it.  My editor was setting tabs on RETURN while the rest of the document is spaces.